docassemble/ALWeaver/data/questions/template_validation.yml

```diff
@@ -585,37 +612,36 @@ subquestion: |
   
   All of the fields in your DOCX ${ interview.uploaded_templates.as_noun("file") } are listed
   in the table below. Confirm that the table looks right.
-  
-  A **bold** name means that this field is **reserved**.
-  Reserved fields are fields that will be handled with a question that
-  comes from the AL Question Library. For example:
-  
-  * Name fields
-  * Address fields
-  * Basic contact information and court information
-  
-  If you expect a field to be bold but it isn't, check the field's spelling.
```

Comment:
Not sure how often this is used, but it does seem like useful info for newbies. Specifically listing out what could be handled by our stuff and how to continue if their form isn't as expected.

Reply:
It was a bit confusing as is, and wasn't clear where this information would have the right context. We could certainly link to it but I think too much detail to put before the table itself